packages/taraxa-ui/src/components/Header/Header.tsx

```diff
@@ -37,17 +37,19 @@ function Header({
       <AppBar {...props} variant='elevation'>
         <Container maxWidth={false}>
           <Toolbar variant='regular'>
-            {Icon && (
-              <a className={classes.headerIconContainer} href='/'>
-                <Icon />
-              </a>
-            )}
+            <div className={classes.flexContainer}>
```

Comment:
Replace this div with:
`<Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>`